build.sh

```diff
@@ -16,6 +16,7 @@ export MINGW_BRANCH=${MINGW_BRANCH:-woarm64}
 
 export RUN_BOOTSTRAP=${RUN_BOOTSTRAP:-1}
 export UPDATE_SOURCES=${UPDATE_SOURCES:-1}
+export RESET_SOURCES=${RESET_SOURCES:-0}
```

Comment:
it is not needed anymore.
RESET_SOURCES is zero by default.
.github/scripts/config.sh

Reply:
It's needed when one wants to locally change it while using this file.